docker-compose.yml

```diff
@@ -1,59 +1,54 @@
 data:
+  container_name: 'data'
```

Comment:
We are using `docker-compose` it is a bad idea to set predefined container name. If you are working on several projects for example.